test/opt/work/staging/fp/receiptProcessing/receiptProcessing_test.csv

```diff
@@ -0,0 +1 @@
+Urszula Piasta-Mansfield,65.5,5/20/2014,/sites/BSC/C5/AmericanIndianProgram,testUnit.pdf,ump4,12345,10000
```

Comment:
I don't think you want to commit these two test files. I tried to update the .gitignore file recently to ignore the test/ directory. My first pass failed, but I think my second was good and Nancy just merged the PR. Maybe if you rebase, or just try tweaking your local file based on my changes this will be cleared up. If there is still a problem with my .gitignore file, please let me know.